tap_github/repository_streams.py

```diff
@@ -1907,6 +1907,457 @@ def query(self) -> str:
+        return params
+
+    @property
+    def query(self) -> str:
+        """Return dynamic GraphQL query."""
+        # Graphql id is equivalent to REST node_id. To keep the tap consistent, we rename "id" to "node_id" and "databaseId" to "id".  # noqa: E501
+        return """
+        query DiscussionsComments($repo: String!, $org: String!, $nextPageCursor_0: String, $nextPageCursor_1: String) {
+        repository(name: $repo, owner: $org) {
+            discussions(first: 10, orderBy: {field: UPDATED_AT, direction: DESC}, after: $nextPageCursor_0) {
+            pageInfo {
+                hasNextPage_0: hasNextPage
+                startCursor_0: startCursor
+                endCursor_0: endCursor
+            }
+            nodes {
+                comments(first: 100, after: $nextPageCursor_1) {
+                pageInfo {
+                    hasNextPage_1: hasNextPage
+                    startCursor_1: startCursor
+                    endCursor_1: endCursor
+                }
+                nodes {
+                    node_id:id
+                    id:databaseId
+                    discussion {
+                    node_id: id
+                    number
+                    }
+                    author {
+                    ... on User {
+                        node_id: id
+                        id: databaseId
+                        login
+                        avatar_url: avatarUrl
+                        html_url: url
+                        type: __typename
+                        site_admin: isSiteAdmin
+                    }
+                    }
+                    author_association: authorAssociation
+                    body
+                    body_html: bodyHTML
+                    body_text: bodyText
+                    created_at: createdAt
+                    published_at: publishedAt
+                    last_edited_at: lastEditedAt
+                    updated_at: updatedAt
+                    created_via_email: createdViaEmail
+                    deleted_at: deletedAt
+                    includes_created_edit: includesCreatedEdit
+                    is_answer: isAnswer
+                    is_minimized: isMinimized
+                    minimized_reason: minimizedReason
+                    upvote_count: upvoteCount
+                    html_url: url
+                    resource_path: resourcePath
+                    editor {
+                    ... on User {
+                        node_id: id
+                        id: databaseId
+                        login
+                        avatar_url: avatarUrl
+                        html_url: url
+                        type: __typename
+                        site_admin: isSiteAdmin
+                    }
+                    }
+                    replies(first: 10) {
+                    nodes {
+                        comment_id: id
+                        replyTo {
+                        replyTo_id: id
```

Comment:
Typo:
```suggestion
                        reply_to_id: id
```
Not sure if we're able to rename the line above as well (is `reply_to: replyTo {` valid syntax?).

Remind me again - what is the relationship between the comment_id and the reply_to_id? If either one is just repeating the id/node_id from [up above](https://github.com/MeltanoLabs/tap-github/pull/376/files#diff-06dc9c6115cbc069ce355913de0c101fedf6956d6f6b4873c5112434596934d3R2234) we can skip it IMO.